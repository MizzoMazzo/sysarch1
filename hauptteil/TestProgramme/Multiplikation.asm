addiu $1, $0, 123
addiu $2, $0, 456
multu $1, $2
mflo $3
mfhi $4