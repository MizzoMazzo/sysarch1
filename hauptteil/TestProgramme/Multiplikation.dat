0x2401007b
0x240201c8
0x00220019
0x00001812
0x00002010
