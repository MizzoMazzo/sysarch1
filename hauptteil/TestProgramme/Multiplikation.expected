@1 7b
@2 1c8
@3 db18
@4 0
